channeldb/invoices.go

```diff
@@ -362,6 +362,30 @@ type Invoice struct {
 	HodlInvoice bool
 }
 
+// HTLCSet returns the set of accepted HTLCs belonging to an invoice. Passing a
+// nil setID will return all accepted HTLCs in the case of legacy or MPP.
+// Otherwise, the returned set will be filtered by the populated setID which is
```

Comment:
comment is not clear I think what it returns in case nil is passed and the invoice is AMP.